<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qPreferences</class>
 <widget class="QDialog" name="qPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qAndora - Preferences</string>
  </property>
  <widget class="QTabWidget" name="prefWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalSettings">
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QFrame" name="userFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="userLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;b&gt;Change User:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="logoutButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="qualityFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="qualityLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;b&gt;Audio Quality:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="qualityBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="notificationFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="notificationLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;b&gt;Desktop Notifications:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="notificationBox">
      <property name="geometry">
       <rect>
        <x>241</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="keySettings">
    <attribute name="title">
     <string>Hotkeys</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="filterSettings">
    <attribute name="title">
     <string>Filters</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
